recipe/meta.yaml

```diff
@@ -1,9 +1,9 @@
-{% set version = "2.2.0.0" %}
+{% set version = "2.3.0.6" %}
 
 {% if cuda_compiler_version in (None, "None", True, False) %}
 {% set cuda_major = 0 %}
 {% else %}
-{% set cuda_major = environ.get("cuda_compiler_version", "11.2").split(".")[0] | int %}
```

Comment:
Are we dropping CUDA 11 support altogether?

Reply:
Yes. No cuda11 support for cutensor versions > 2.2